UEVR/MainWindow.xaml.cs

```diff
@@ -637,6 +753,22 @@ public partial class MainWindow : Window {
             return null;
         }
 
+        private void RemoveVRPlugins(string? pluginsPath) {
```

Comment:
Maybe not delete them outright. Add a _bak to the original folder names or something along these lines so the user doesn't have to verify game integrity to get a clean install back.

Reply:
Whole point of this change is to make UEVR approachable for non technical people who won't know how to go and rename files inside game folder. And for the technical ones it's still easier to verify game files because it just works. 
There's also curious case of some Gamepass games where I can delete or execute files but not open them in editor or drop files into game folder so I guess DRM is blocking access permissions. 

But if you think it's important I can change it.